controller/AdminAccessController.php

```diff
@@ -109,23 +113,16 @@ public function savePermissions(): void
         $recursive = ($this->getRequest()->getParameter('recursive') === '1');
 
         try {
-            $this->validateCsrf();
-
-            $service = $this->getPermissionService();
```

Comment:
`getPermissionService()` method may be removed.

Reply:
Removed